scipy/sparse/linalg/_eigen/lobpcg/lobpcg.py

```diff
@@ -25,7 +25,7 @@
 from scipy.sparse import isspmatrix
 from numpy import block as bmat
 
-__all__ = ["lobpcg"]
+__all__ = ["lobpcg", "_b_orthonormalize"]
```

Comment:
Do we really want to expose this function? I suppose you only did that to make the import work in tests?

Reply:
Yes

Reply:
Why do we want this exposed higher up in the API?

Reply:
I do not know how to do it properly, just asked ChatGPT :-)

Reply:
Please don't. We have already difficulties with licensing, so until legal aspect of such things are solved (which could take years...), we ask everyone to "fully own, author and be accountable" for their code.

Of course, what you do in the background, we cannot know if you don't tell anyone. But if such assistant produce "by accident" some GPL code (which it is doing from time to time as showed multiple times), then we are in (big) trouble if this is not detected during a review and get's released.

Instead of 
`from scipy.sparse.linalg._eigen.lobpcg import _b_orthonormalize`
this should work
`from scipy.sparse.linalg._eigen.lobpcg.lobpcg import _b_orthonormalize`

Reply:
It was a joke about  ChatGPT, so no worries.

Reply:
haha ok 😅 Internet+reading is hard with jokes